### PR TITLE
Added animations support in Angular 2 RC3

### PR DESCRIPTION
**Pull request by tzraikov**

### CODE REVIEW

File: nativescript-angular/animation-player.ts [hunk 1]
```diff
@@ -0,0 +1,304 @@
...
+        if (this.animation && this.animation.isPlaying) {
+            this.animation.cancel();
+        }
+    }
+
+    finish(): void {
+        if (this.animation && this.animation.isPlaying) {
+            this.animation.cancel();
+        }
+        // TODO: apply end values
```
Comment (vakrilov) [inline nativescript-angular/animation-player.ts]: TODO: apply end values ?

File: nativescript-angular/animation-player.ts [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+    }
+    reset(): void {
+        if (this.animation && this.animation.isPlaying) {
+            this.animation.cancel();
+        }
+        // TODO: apply start values
```
Comment (vakrilov) [inline nativescript-angular/animation-player.ts]: TODO: apply end values ?

File: nativescript-angular/animation-player.ts [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+                }
+            }
+            
+            this.animation.play()
+                .then(() => { this._onFinish(); })
+                .catch((e) => {});
+        }
+    }
+
+    pause(): void {
```
Comment (vakrilov) [inline nativescript-angular/animation-player.ts]: If pause is not supported maybe throw exception?

File: nativescript-angular/animation-player.ts [hunk 4]
```diff
@@ -0,0 +1,304 @@
...
+        this.play();
+    }
+
+    destroy(): void {
+        this.reset();
+        this._onFinish();
+    }
+
+    setPosition(p: any): void {
+
```
Comment (vakrilov) [inline nativescript-angular/animation-player.ts]: Throw exception?

**Comment by vakrilov** [comment]
Couple of comments about the examples inside `ng-sample`:
- `nativeScriptBootstrap(AnimationKeyframesTest)` - doesn't show anything - both iOS and Android
- `nativeScriptBootstrap(AnimationEnterLeaveTest)` - taping on on of the items causes a crash - both iOS and Android
- `nativeScriptBootstrap(AnimationEnterLeaveTest)` - the `enter` animation doesn't seem to trigger on iOS
- `nativeScriptBootstrap(AnimationStatesTest);` - on Android - when triggering the animation the button background changes to white first and then form white to red/green

**Comment by tzraikov** [comment]
run ci

**Comment by vakrilov** [comment]
👍 
brace

**Pull request merged**